lib/org/chromium/webidl/PipelineMessage.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
Passing this whole thing around seems excessive. Can't each stage in the pipeline have clearly defined input and output messages?